apps/meteor/tests/end-to-end/api/00-miscellaneous.js

```diff
@@ -389,6 +389,59 @@ describe('miscellaneous', function () {
 				})
 				.end(done);
 		});
+
+		before((done) => {
+			const teamName = `new-team-name-${Date.now()}`;
+			request
+				.post(api('teams.create'))
+				.set(credentials)
+				.send({
+					name: teamName,
+					type: 0,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((res) => {
+					expect(res.body).to.have.property('success', true);
+					expect(res.body).to.have.property('team');
+					expect(res.body).to.have.nested.property('team._id');
+				})
+				.end(done);
+		});
+
+		it('should return an object containing rooms and totalCount from teams', (done) => {
+			request
+				.get(api('directory'))
+				.set(credentials)
+				.query({
+					query: JSON.stringify({
+						text: '',
+						type: 'teams',
+					}),
+					sort: JSON.stringify({
+						name: 1,
+					}),
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((res) => {
+					expect(res.body).to.have.property('result');
+					expect(res.body.result).to.be.an(`array`);
+					expect(res.body).to.have.property('total');
```

Comment:
ideally, you should also check if the values returned by the endpoint are the expected ones.. for example, if you have created just a single team but it returns `total: 2`, it probably means it is returning not only teams, so there is probably a bug on the endpoint.
```suggestion
					expect(res.body).to.have.property('total', 1);
```